code/modules/mob/living/carbon/human/species_types/snail.dm

```diff
@@ -82,6 +83,7 @@
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	//SKYRAT EDIT BEGIN - Roundstart Snails
 	slowdown = 6 // The snail's shell is what's making them slow.
```

Comment:
This kind of addition to an item should be modularized, I don't see why it had to be done non-modularly.

Reply:
that was you senpai,,